serviceregistry/src/main/java/eu/arrowhead/core/serviceregistry/ServiceRegistryController.java

```diff
@@ -1068,31 +1077,31 @@ public void removeServiceInterface(@PathVariable(value = PATH_VARIABLE_ID) final
 	// assistant methods
 	
 	//-------------------------------------------------------------------------------------------------
-	private SystemResponseDTO callCreateSystem(final SystemRequestDTO request, final String origin) {
+	private SystemResponseDTO callCreateSystem(final HttpServletRequest servletRequest, final SystemRequestDTO dto, final String origin) {
 		logger.debug("callCreateSystem started...");
 		
-		checkSystemRequest(request, origin, true);
+		checkSystemRequest(servletRequest, dto, origin, true);
 		
-		final String systemName = request.getSystemName().toLowerCase().trim();
-		final String address = request.getAddress().toLowerCase().trim();
-		final int port = request.getPort();
-		final String authenticationInfo = request.getAuthenticationInfo();
-		final Map<String,String> metadata = request.getMetadata();
+		final String systemName = dto.getSystemName().toLowerCase().trim();
+		final String address = dto.getAddress().toLowerCase().trim();
+		final int port = dto.getPort();
+		final String authenticationInfo = dto.getAuthenticationInfo();
+		final Map<String,String> metadata = dto.getMetadata();
 		
 		return serviceRegistryDBService.createSystemResponse(systemName, address, port, authenticationInfo, metadata);
 	}
 	
 	//-------------------------------------------------------------------------------------------------
-	private SystemResponseDTO callUpdateSystem(final SystemRequestDTO request, final long systemId) {
+	private SystemResponseDTO callUpdateSystem(final HttpServletRequest servletRequest, final SystemRequestDTO dto, final long systemId) {
```

Comment:
I don't see where do you use servletRequest parameter here

Reply:
that's a mistake